### PR TITLE
DE43073/pass empty string instead of none grade.

### PR DESCRIPTION
**Pull request by mbosiljevac**
Note This PR also has to update the grade-result

### CODE REVIEW

File: components/helpers/rubricGradeSyncHelpers.js [hunk 1]
```diff
@@ -10,13 +10,17 @@ export function mapRubricScoreToGrade(rubricInfo, currentGrade, newScore) {
...
+	if (currentGrade.scoreType === GradeType.Letter && currentGrade.letterGradeOptions) {
 		const percentage = (newScore / outOf) * 100;
-		const map = currentGrade.entity.properties.letterGradeSchemeRanges;
-		for (const [key, value] of Object.entries(map)) {
-			if (percentage >= value) {
-				letterGrade = key;
-				break;
+		const map = currentGrade.letterGradeOptions;
+		let currentPercentStart = 0;
+		for (const letterGradeOption of Object.values(map)) {
```
Comment (jeroach) [inline components/helpers/rubricGradeSyncHelpers.js:17]: Sorry, could you please explain what this block is doing? I'm confused :)

Comment (jeroach) [reply components/helpers/rubricGradeSyncHelpers.js:17]: Ah I see, thanks :)

Comment (mbosiljevac) [reply components/helpers/rubricGradeSyncHelpers.js:17]: no worries :)

**Review by jeroach — APPROVED** [review]
LGTM

**Review by jmurray-d2l — APPROVED** [review]
Nice work!

**Pull request merged**